psignifit/psigniplot.py

```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
 from typing import Union, List
 
+import warnings
```

Comment:
nit: the import should be with the standard library import block (one line up)

Reply:
out of curiosity, is there a guideline for that? order of imports? I never know.

Reply:
yes, from PEP8:
"""

Imports should be grouped in the following order:

- Standard library imports.
- Related third party imports.
- Local application/library specific imports.

You should put a blank line between each group of imports.
"""

Reply:
thank you darling.